src/Microsoft.VisualStudio.ProjectSystem.CSharp/ProjectSystem/LanguageServices/CSharpCommandLineParserProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
+{
+    internal class CSharpCommandLineParserProvider
+    {
+        [ImportingConstructor]
```

Comment:
Does this still need to be here even if it's an empty constructor? Can't we just remove the constructor entirely?